Engine/GraphicsCore.cs

```diff
@@ -127,19 +133,26 @@ public static void Init(IntPtr HWND, int width, int height)
             AssetsManager.LoadShader("particles_init",              "BaseAssets\\Shaders\\Particles\\particles_init.csh");
             AssetsManager.LoadShader("particles_update_energy",     "BaseAssets\\Shaders\\Particles\\particles_update_energy.csh");
             AssetsManager.LoadShader("particles_update_physics",    "BaseAssets\\Shaders\\Particles\\particles_update_physics.csh");
+            AssetsManager.LoadShader("screen_quad",                 "BaseAssets\\Shaders\\screen_quad.vsh");
 
             AssetsManager.LoadShaderPipeline("volume", Shader.Create("BaseAssets\\Shaders\\VolumetricRender\\volume.vsh"),
                                                        Shader.Create("BaseAssets\\Shaders\\VolumetricRender\\volume.fsh"));
 
-            Shader screenQuadShader = Shader.Create("BaseAssets\\Shaders\\screen_quad.vsh");
+            Shader screenQuadShader = AssetsManager.Shaders["screen_quad"];
             AssetsManager.LoadShaderPipeline("deferred_light_point", screenQuadShader, Shader.Create("BaseAssets\\Shaders\\DeferredRender\\deferred_light_point.fsh"));
             AssetsManager.LoadShaderPipeline("deferred_light_directional", screenQuadShader, Shader.Create("BaseAssets\\Shaders\\DeferredRender\\deferred_light_directional.fsh"));
             AssetsManager.LoadShaderPipeline("deferred_addLight", screenQuadShader, Shader.Create("BaseAssets\\Shaders\\DeferredRender\\deffered_addLight.fsh"));
             AssetsManager.LoadShaderPipeline("deferred_gamma_correction", screenQuadShader, Shader.Create("BaseAssets\\Shaders\\DeferredRender\\deferred_gamma_correction.fsh"));
 
+            bloomEffect = new PixelShaderBloom(width, height, 5);
+            OnResized += bloomEffect.Resize;
```

Comment:
Should not be here, either subscribe to event from constructor of bloomEffect, or just use the info from texture you pass to the effect when calling Process as i mentioned before

Reply:
Resized method was revomed from PostProcessEffect